### ISSUE TITLE
got 'warning: changing a uncontrolled input' when updated react to v15.0.0

### ISSUE THREAD

**Issue by smilingpoplar** [erikras/redux-form#785]
Updated react to v15.0.0 today, and I got this warning: 
```
Warning: <Component> is changing a uncontrolled input of type text to be controlled. Input 
elements should not switch from uncontrolled to controlled (or vice versa). Decide between 
using a controlled or uncontrolled input element for the lifetime of the component. More info: 
https://fb.me/react-controlled-components
```

And it seems relate to [the initial value of the \<input>'s `value` attribute](https://github.com/facebook/react/issues/6222)

**Comment by erikras** [comment]
Duplicate of #735. Known problem, thanks.

**Comment by erikras** [comment]
React v15 support released with [`redux-form v5.0.0`](https://github.com/erikras/redux-form/releases/tag/v5.0.0).

**Comment by smilingpoplar** [comment]
cooool
